frontend/components/Reports/ScanFinding.jsx

```diff
@@ -25,15 +25,15 @@ const ScanFinding = ({
     ({
       name: title, solution, description,
     } = finding);
-    anchor = `finding-${finding.alertRef}`;
+    anchor = `finding-${finding.alertRef}${ignoreSource ? '-suppressed' : ''}`;
     count = parseInt(finding.count, 10);
     locations = finding.instances || [];
     references = finding.referenceURLs || [];
     hasMoreInfo = finding.otherinfo;
   }
   if (scanType === 'a11y') {
     title = `${finding.help}.`;
-    anchor = `finding-${finding.id}`;
+    anchor = `finding-${finding.id}${ignoreSource ? '-suppressed' : ''}`;
```

Comment:
I'd condition this on `ignore` rather than the source (since it's false for non-ignored items and custom ignored items)